modules/jpa/src/main/java/io/oasp/module/jpa/common/api/to/SearchCriteriaTo.java

```diff
@@ -21,7 +21,7 @@
   /** @see #getPagination() */
   private PaginationTo pagination;
 
-  /** @see getSearchTimeout */
+  /** @see #getSearchTimeout */

```

Comment:
Still wrong reference syntax has to be `#getSearchTimeout()`. Please consult JavaDoc manual. Does your IDE not complain about this? I do get warnings in such case. Something seems wrong here...